doc/sym1.sgml

```diff
@@ -0,0 +1,133 @@
+<!doctype linuxdoc system>
+
+<article>
+<title>Synertek Systems Sym-1 specific information for cc65
+<author><url url="mailto:wayne@parhamdata.com" name="Wayne Parham">
+
+<abstract>
+An overview over the Sym-1 runtime system as it is implemented for the cc65 C compiler.
+</abstract>
+
+<!-- Table of contents -->
+<toc>
+
+<!-- Begin the document -->
+
+<sect>Overview<p>
+
+This file contains an overview of the Sym-1 runtime system as it comes with the cc65 C compiler.  It describes the memory layout, Sym-1 specific header files, available drivers, and any pitfalls specific to the platform.
+
+Please note that Sym-1 specific functions are just mentioned here, they are described in detail in the separate <url url="funcref.html" name="function reference">. Even functions marked as "platform dependent" may be available on more than one platform.  Please see the function reference for more information.
+
+<sect>Binary format<p>
+
+The output format generated by the linker for the Sym-1 target is a raw binary BIN file, which is essentially a memory image.  You can convert this to a HEX file using BIN2HEX, which is a popular open-source conversion utility program.  A HEX file has ASCII representations of the hexadecimal byte values of the machine-language program.
```

Comment:
Thanks for adding content here. However, I didn't think about an explanation of BIN and HEX files. I was thinking about what do to with the file generated by the linker in order to get it run. If getting a HEX file is part of the at process, it's enough to say something like "Use e.g. BIN2HEX to generate a HEX file." But the interesting part is what to do wit the HEX file!

Reply:
Yes, I understand.  I've added comments telling the user how to upload their HEX file onto the Sym-1.